### DISCUSSION TITLE
[Help wanted] `__VERSION__`

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vuepress/core#214] [Q&A] [answered]
Is something special that vuepress will change `__VERSION__` in to under vite?

![image](https://user-images.githubusercontent.com/33315834/121811232-5ea58a80-cc96-11eb-89c0-325146dc0109.png)

The code is still working fine under webpack, but getting errors in vite. Vite is reporting a error inside a string, That's very werid, so I guess it may be changed to something else in vite? Which is acutally breaking the string to pieces?

If so, how can I avoid it?

**Comment by yzx9** [comment, marked as answer]
Looks like the issue I had earily, looks at #143 might help you.

**Reply by Mister-Hope** [reply]
Thanks for that🥰

But I cannot see a possible solution here, because it's a third party code.😅

So I fully see why vite is very fast😕, by dropping a lot of steps and checks which tools like webpack are doing.👎

**Reply by yzx9** [reply]
haha, so it does, you can`t make an omelet without breaking eggs 😥
vite is not perfect but so faaaaast 🚀
